src/main/java/edu/kis/powp/jobs2d/command/OperateToCommand.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class OperateToCommand implements DriverCommand {
+
+    private final Job2dDriver driver;
+    private int x;
+    private int y;
+
+    public OperateToCommand(int posX, int posY, Job2dDriver driver) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).